### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Paula-Kli**
it is the same issue as https://github.com/fsr-de/myHPI/pull/30

The main changes are the updates of wagtail and Django. But also of Bleach.

### CODE REVIEW

**Comment by jeriox** [comment]
@Paula-Kli you can manually update wagtail-localize in pyproject.toml since we are several versions behind

**Comment by jeriox** [comment]
If you don't want to search for the newest version, you can also run `poetry add wagtail-localize@latest` which will update the entry in pyproject.toml

File: poetry.lock [hunk 1]
```diff
@@ -409,7 +424,7 @@ python-versions = "*"
 name = "isort"
 version = "5.10.1"
 description = "A Python utility / library to sort Python imports."
-category = "dev"
+category = "main"
```
Comment (Paula-Kli) [inline poetry.lock:427]: for example here I changed the dependency from dev to main by accident but don't know how to change it back except for deleting isort and installing it new... does anybody have a good idea on how to change it back? I can not do it manually as poetry computes a hash over that file

**Review by jeriox — APPROVED** [review]

**Comment by jeriox** [comment]
works for me as well, thx

**Pull request merged**